R/join_keys.R

```diff
@@ -0,0 +1,155 @@
+#' Manage relationships between datasets using `join_keys`
+#'
+#' @description
+#' `join_keys()` facilitates the creation and retrieval of relationships between datasets.
+#' `join_keys` class extends a list and contains keys connecting pairs of datasets. Each element
+#' of the list contains keys for specific dataset. Each dataset can have a relationship with
+#' itself (primary key) and with other datasets.
+#'
+#' Note that `join_keys` list is symmetrical, that is, when keys are set between `dat1` and `dat2` it
+#' is automatically mirrored between `dat2` and `dat1`.
+#'
+#' @details
+#'
+#' - `join_keys()`: Returns an empty `join_keys` object when called without arguments.
+#' - `join_keys(x)`: Returns the `join_keys` object contained in `x` (if it contains one).
```

Comment:
```suggestion
#' - `join_keys(x)`: Returns the `join_keys` object contained in `x`, if any.
```
Otherwise empty `join_keys` or `NULL`?